spec/lib/appsignal/integrations/mongo_ruby_driver_spec.rb

```diff
@@ -81,7 +81,7 @@
         transaction.should receive(:finish_event).with(
           'query.mongodb',
           'find | test | SUCCEEDED',
-          "{\"foo\":\"?\"}",
+          instance_of(Appsignal::Extension::Data),
```

Comment:
Aha, even if we create exactly the same data object it's comparison is done on pointer level and not values. It's probably not easy to make a comparison method for the internal values.

Can we move the conversion to the `Data` object later in the process? When it's received by `finish_event` for example

Or maybe even better: wrap it in an event formatter that converts all hashes and arrays? Rather than let individual integrations deal with the specifics of creating a valid body